domain/src/main/java/org/oppia/android/domain/state/StateGraph.kt

```diff
@@ -33,6 +37,10 @@ class StateGraph constructor(
       outcome.missingPrerequisiteSkillId.isNotEmpty() ->
         answerOutcomeBuilder.missingPrerequisiteSkillId = outcome.missingPrerequisiteSkillId
       outcome.destStateName == currentState.name -> answerOutcomeBuilder.sameState = true
+      !outcome.labelledAsCorrect &&
+        outcome.feedback.contentId.contains("feedback", true) &&
```

Comment:
Why is this needed? I'm very much not keen on hardcoding content IDs into code as that's always going to be potentially fragile.